packages/hosted-assets/Kaizen/HostedAssets.elm

```diff
@@ -17,3 +21,23 @@ assetUrl "some/blob.png" -> "<https://<origin>/some/blob.png">
 assetUrl : String -> String
 assetUrl path =
     [ originBaseUrl, path ] |> String.join "/"
+
+
+type Role
+    = Presentation
```

Comment:
The code that determines how the `img` is rendered is based on https://github.com/cultureamp/kaizen-design-system/blob/master/packages/component-library/components/Icon/Icon.elm. Only the `Presentation` variant has been implemented. The `Img` variant can be added if a need arises.